tests/unit/test_flist.py

```diff
@@ -82,30 +87,60 @@ def test_api_foldLeft(self):
     self.assertEqual(120.0, alist.foldLeft(1.0, lambda a, e: a * e))
 
   def test_api_foldLeft_with_short_stop(self):
+    """Testing API Cons.foldLeft with arg short_stop_func."""
+    # TODO: fpu/57 - pytest parameterize not supported in unittest class
```

Comment:
Nit: If it makes sense to you, let's use Google style to make todo marker. e.g.:
```
# TODO(#57): pytest parameterize not supported in unittest class
```
FYI